test/sequential/test-benchmark-http.js

```diff
@@ -28,7 +31,7 @@ const child = fork(runjs, ['--set', 'benchmarker=test-double',
                            '--set', 'len=1',
                            '--set', 'n=1',
                            'http'],
-                   {env: {NODEJS_BENCHMARK_ZERO_ALLOWED: 1}});
+                   {env: env});
```

Comment:
No (at least the linter passes without).

I didn't realise you could use the shorthand props on Node 4, I do think it's nicer, will update.